# This file is generated by
# $ rosrun camera_calibration cameracalibrator.py

image_width: 1504
image_height: 1504
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [753.86903,   0.     , 829.59748,
           0.     , 703.23599, 918.33654,
           0.     ,   0.     ,   1.     ]
camera_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.287251, 0.041911, -0.039052, -0.022004, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [678.98792,   0.     , 847.68605,   0.     ,
           0.     , 621.01874, 874.74736,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]
